AGENTS.md

```
@@ -285,8 +285,9 @@ GitHub Actions workflows test across:
 Key workflows:
 
 - `.github/workflows/test-php.yml` - PHP testing
+- `.github/workflows/release-php.yml` - GitHub release on a tag
 [//]: # (#;< PHP_PHAR)
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)